TLSharp.Core/Network/MtProtoSender.cs

```diff
@@ -305,6 +305,12 @@ private bool HandleRpcResult(ulong messageId, int sequence, BinaryReader message
                     var dcIdx = int.Parse(resultString);
                     throw new UserMigrationException(dcIdx);
                 }
+		else if (errorMessage.StartsWith("NETWORK_MIGRATE_"))
```

Comment:
missing one space in this line (indentation incorrect)

Reply:
Hmm, can't see it in my original file( Looks like a github bug.
![image](https://user-images.githubusercontent.com/9249545/30775040-79611e44-a09d-11e7-8fac-3bf05b65255b.png)

Reply:
haha, no, there's no github bug here, see it in raw: https://github.com/sochix/TLSharp/commit/a2322b6ce433b21ed648941683d96ed0caaa5ab7.patch (you may have a misconfigured editor that hides you differences between tabs and spaces)